lib/filestorage.js

```diff
@@ -144,6 +137,14 @@ class FileStorage {
   [getFilepath](id) {
     return utils.getFilepath(this.dir, common.idToPath(id));
   }
+
+  [getDataStats](data, checksum, dedupHash) {
```

Comment:
I don't think this needs to be on FileStorage, does it?
I think it can just be a separate function.